config.py

```diff
@@ -103,33 +98,26 @@ class GG(): pass
 SCREENSHOT_MODE=is_sdk and False
 game_name="Adventure Land"
 appengine_id="twodimensionalgame"
-live_domain='adventure.land'
-sdk_domain='thegame.com'
+live_domain=secrets.base_url
+sdk_domain=secrets.base_url
+
 SDK_UPLOAD_PASSWORD=ELEMENT_PASSWORD=secrets.sdk_password
 
 def gdi(self=None):
 	domain=GG()
 	if is_sdk:
-		if self:
-			url=urlparse(self.request.url)
-			protocol=url.scheme
-			hostname=url.hostname
-		else:
-			protocol="http"
-			hostname=sdk_domain
-
-		domain.base_url=protocol + "://" + hostname
-		domain.pref_url=domain.base_url
+		domain.base_url=self and "http://%s"%self.request.headers.get("Host") or "http://%s.%s"%(sdk_domain[1],sdk_domain[2])
+		domain.pref_url=self and "http://%s"%self.request.headers.get("Host") or "http://%s.%s"%(sdk_domain[1],sdk_domain[2])
 		domain.server_ip="192.168.1.125"
 		domain.stripe_pkey=stripe_pkey
 		domain.stripe_enabled=False
 		domain.https_mode=False
-		domain.domain=hostname
+		domain.domain=self and ["www",self.request.headers.get("Host").split(".")[0],self.request.headers.get("Host").split(".")[1]] or sdk_domain
 	else:
 		protocol="http"
 		if self and "https" in (self.request.headers.get("Cf-Visitor") or ""): protocol="https"
-		domain.base_url=protocol + "://" + live_domain
-		domain.pref_url=domain.base_url
+		domain.base_url="%s://%s.%s"%(protocol,live_domain[1],live_domain[2])
+		domain.pref_url="https://%s.%s"%(live_domain[1],live_domain[2])
```

Comment:
This should use the code from main, it supports running in development mode as well as production. 
The code here seems like older, previous code.